scripts/gha/desktop_tester.py

```diff
@@ -40,6 +40,10 @@
 
 flags.DEFINE_string("testapp_dir", None, "Look for testapps in this directory.")
 flags.DEFINE_string("testapp_name", "integration_test", "Name of the testapps.")
+flags.DEFINE_string(
+    "artifact_name", "",
+    "Create test log artifact is test-results-$artifact_name.log."
```

Comment:
nit: remove `is`

Reply:
desktop_tester referring to "artifact" is weird - maybe this parameter should be "logfile"?